src/diffusers/pipelines/blip_diffusion/pipeline_blip_diffusion_controlnet.py

```diff
@@ -0,0 +1,375 @@
+# Copyright 2023 Salesforce.com, inc.
```

Comment:
Should we have it inside the `controlnet` module and rename it to `pipeline_controlnet_blip_diffusion.py`? Not too strongly opinionated about this either since BLIP Diffusion utilizes Stable Diffusion parameters anyway. 

@patrickvonplaten what are your thoughts?

Reply:
Good point, I'd actually put it under the controlnet folder indeed

Reply:
Okay, shifting to controlnet module then

Reply:
Moved